src/consensus/aft/raft.h

```diff
@@ -157,12 +157,12 @@ namespace aft
       state(state_),
       executor(executor_),
       request_tracker(request_tracker_),
+      view_change_tracker(state_->my_node_id, 2, view_change_timeout_),
```

Comment:
We should probably define the initial view as 2 somewhere in this file as it is now used in a quite a few places.